class1.md

```diff
@@ -0,0 +1,261 @@
+
+### How is machine learning applied?
+
+<img align="right" width="150" src="https://media.giphy.com/media/3oEduS0HyWuqR7kXL2/giphy.gif">
+
+Machine learning applications are all around us! Every time Spotify serves up a customized playlist or radio station based on the music that you like, Facebook loads your newsfeed, your phone autocorrects you, or you interact with a voice assitant like Siri, you are interacting with machine learning algorithms.
+
+Every-day applications:
+* Search engines
+* Recommendation systems (Yelp, Netflix, Spotify)
+* Voice assistants
+* Social media feeds
+* Text prediction
+
+#### Machine learning techniques are becoming increasingly important to biological research.
+
+Researchers are developing algorithms that use machine learning to classify histopathological images, visualize and find clusters within single cell datasets, and much more.
+
+Biology specific applications:
+* Prediction of protein secondary structure based on RNA-sequence.
+* Classification of histopathological images of breast tissue to identify malignant tumors.
+* Using gene expression data to classify patients into different clinical groups.
+
+Machine learning work that is happening here at Fred Hutch:
+* Gottardo lab - [automatic gating for flow cytometry data.](https://www.biorxiv.org/content/10.1101/702118v2)
+* Emily Silgard's data science group - [natural language processing to extract information from unstructured electronic medical records notes.](https://www.sciencedirect.com/science/article/abs/pii/S1532046419302217)
+* The Data Visualization Center - [single-cell visualization and clustering of RNA-sequencing data to find rare subtypes of cells that emerge in the mouse embryo.](https://oncoscape.v3.sttrcancer.org/atlas.gs.washington.edu.mouse.rna/landing)
+
+### How do these machine learning applications work?
+
+There are different types of machine learning and each have their own specific applications and requirements.
+
+Four major paradigms in machine learning:
+* **Supervised**
+* **Unsupervised**
+* Semi-supervised
+* Reinforcement learning
+
+The focus of this course will be on supervised and unsupervised learning only, but it’s good to be aware that these are not the only types of machine learning. While increasingly complex machine learning algorithms are highlighted on TV and in the news, it's important to remember that the most commonly implemented form of machine learning is the simplest - supervised machine learning.
+
+### Anatomy of a machine learning problem
+
+The purpose of using machine learning is generally to use data make an inference or prediction. To do this, these algorithms access large datasets with many features. Datasets rarely, if ever, will capture all of the complex underlying variables that result in the Truth that we see in the world. Datasets merely capture the features that we think are related to the system that we are trying to uncover and that we can adequately measure and track.
```

Comment:
I know what you mean by Truth, but I would rephrase (and not capitalize) since it is hard to understand without the context of knowing you (and Thomas) :)